src/webgpu/util/texture/texelData.spec.ts

```diff
@@ -61,19 +73,15 @@ function doTest(
 
   [[block]] struct Output {
     ${rep.componentOrder
-      .map((C, i) => {
-        return `[[offset(${i * 4})]] result${C} : ${shaderType};`;
-      })
+      .map((C, i) => `[[offset(${i * 4})]] result${C} : ${shaderType};`)
       .join('\n')}
```

Comment:
Maybe `pp` should allow interpolants to be arrays of strings that will get joined by `\n` automatically (and maybe even indented properly?)